samples/highcharts/demo/venn-diagram/demo.js

```diff
@@ -1,42 +1,128 @@
+        }, {
+            sets: ['Dashboards'],
+            value: 3,
+            dataLabels: {
+                style: {
+                    fontSize: 13
+                }
+            }
         }, {
-            sets: ['Fast'],
-            value: 2
+            sets: ['Gantt'],
+            value: 2.5,
+            dataLabels: {
+                style: {
+                    fontSize: 13
+                }
+            }
         }, {
-            sets: ['Cheap'],
-            value: 2
+            sets: ['Maps'],
+            value: 3,
+            dataLabels: {
+                style: {
+                    fontSize: 13
+                }
+            }
         }, {
-            sets: ['Good', 'Fast'],
+            sets: ['Gantt', 'Maps', 'Stock'],
             value: 1,
-            name: 'More expensive'
+            name: 'Core'
         }, {
-            sets: ['Good', 'Cheap'],
+            sets: ['Stock', 'Core'],
             value: 1,
-            name: 'Will take time to deliver'
+            name: 'DateTime Series and Axis'
         }, {
-            sets: ['Fast', 'Cheap'],
+            sets: ['Gantt', 'Core'],
             value: 1,
-            name: 'Not the best quality'
+            name: 'X-range Series and DateTime Axis'
         }, {
-            sets: ['Fast', 'Cheap', 'Good'],
+            sets: ['Maps', 'Core'],
             value: 1,
-            name: 'They\'re dreaming'
+            name: 'Heatmap and ColorAxis'
+        }, {
+            sets: ['Stock', 'Gantt'],
+            value: 0.25,
+            name: 'Navigator & RangeSelector'
+        }, {
+            sets: ['Dashboards', 'Core'],
+            value: 1,
+            name: 'Data Layer'
+        }, {
+            sets: ['Dashboards', 'DataGrid'],
+            value: 0.5,
+            name: 'DataGrid'
+        }, {
+            sets: ['Dashboards', 'KPI'],
+            value: 0.5,
+            name: 'KPI'
+        }, {
+            sets: ['DataGrid'],
+            value: 0.5,
+            name: ''
+        }, {
+            sets: ['KPI'],
+            value: 0.2,
+            name: 'KPI'
+        }, {
+            sets: ['Custom'],
+            value: 2,
+            name: 'Custom component'
+        }, {
+            sets: ['Custom', 'Dashboards'],
+            value: 0.3,
+            name: 'Sync API'
         }]
     }],
     title: {
-        text: 'The Unattainable Triangle'
+        text: 'Highsoft products relationships'
+    },
+    subtitle: {
+        text: 'Highcharts Core, Stock, Maps, Gantt, and Dashboards'
+    },
+    accessibility: {
+        description: 'This Venn diagram shows the relationships between ' +
+            'different Highsoft products, including ' +
+            'Highcharts Core, Stock, Maps, Gantt and Dashboards',
+        point: {
+            valueDescriptionFormat: '{point.name}'
+        },
+        series: {
+            describeSingleSeries: true,
+            descriptionFormat: '{series.name} with ' +
```

Comment:
`{series.name} `is currently named Series 1 since series does not have a name which gives us the text "Series 1 with 17 relations." Naming of the series can be improved